test/shared/src/main/scala/zio/test/TestOutput.scala

```diff
@@ -18,7 +20,8 @@ object TestOutput {
     ZLayer.fromZIO(
       for {
         executionEventPrinter <- ZIO.service[ExecutionEventPrinter]
-        outputLive            <- TestOutputLive.make(executionEventPrinter)
+        // If you need to enable the debug output to diagnose flakiness, set this to true
+        outputLive <- TestOutputLive.make(executionEventPrinter, debug = false)
```

Comment:
We already have a way to configure, for example, the size of generators. Perhaps we should add the flag there, generalizing that mechanism so you can configure test settings in a uniform way?

cc @adamgfraser ?

Reply:
I think that gets added at a lower level, basically configuring aspects of how individual tests operate rather than configuring the test framework itself.

I also think we probably need to do a little more use of this feature internally and make sure it is helpful for us in diagnosing CI flakiness before rolling it out more broadly.